x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/security/authc/RemoteAccessAuthentication.java

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+package org.elasticsearch.xpack.core.security.authc;
```

Comment:
I'm considering adding a new package `o.e.x.c.s.remote_access` or similar, which could be a better place for this class. Depends on how many more remote-access specific classes we end up with. For now, `o.e.x.c.s.authc` seems like an okay place for it.